ts/features/services/home/components/FeaturedServicesCarousel.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  HSpacer,
+  IOSpacingScale,
+  IOVisualCostants
+} from "@pagopa/io-app-design-system";
+import * as React from "react";
+import { FlatListProps, StyleSheet } from "react-native";
+import { FlatList } from "react-native-gesture-handler";
+import { WithTestID } from "../../../../types/WithTestID";
+import {
+  FeaturedServiceCard,
+  FeaturedServiceCardProps,
+  FeaturedServiceCardSkeleton
+} from "./FeaturedServiceCard";
+
+export type FeaturedServicesCarouselProps = WithTestID<{
+  services: Array<FeaturedServiceCardProps>;
+}>;
+
+const CARD_SPACING: IOSpacingScale = 8;
+const CARD_WIDTH: number = 100;
+const CARD_TOTAL_WIDTH = CARD_WIDTH + CARD_SPACING;
+
+const style = StyleSheet.create({
+  list: {
+    marginHorizontal: -IOVisualCostants.appMarginDefault,
+    paddingHorizontal: IOVisualCostants.appMarginDefault
+  }
+});
+
+const FeaturedServicesCarouselBaseComponent = <T,>({
+  data,
+  ...props
+}: WithTestID<FlatListProps<T>>) => (
+  <FlatList
+    {...props}
+    horizontal={true}
+    ItemSeparatorComponent={() => <HSpacer size={CARD_SPACING} />}
+    data={data}
+    style={style.list}
+    ListFooterComponent={() => <HSpacer size={48} />}
+    showsHorizontalScrollIndicator={false}
+    keyExtractor={(_, index) => `featured_service_card_${index.toString()}`}
+    snapToInterval={CARD_TOTAL_WIDTH}
```

Comment:
As suggested for the other carousel component:
```suggestion
    decelerationRate="fast"
    snapToInterval={CARD_TOTAL_WIDTH}
```